### ISSUE TITLE
How to annotate an existing PDF

### ISSUE THREAD

**Issue by ericbf** [julianhille/MuhammaraJS#290]
Hello! I have a requirement to add underline/strikethrough to existing PDFs. Really, I have some PDFs where I need to underline all text, and I have other PDFs where I need to add strikethrough to all text. Is there any way to do that?

Related to #283

**Comment by julianhille** [comment]
so as example just a simple pdf file and then underline or strikethrough ALL the text?

**Comment by ericbf** [comment]
Yeah, that’s essentially what I’m looking for. I was trying to use the annot on a recipe, but couldn’t figure it out.
